frontend/src/component/project/Project/ProjectSettings/Settings/DeleteProject.tsx

```diff
@@ -36,41 +36,55 @@ export const DeleteProject = ({
     const automatedActionsEnabled = useUiFlag('automatedActions');
     const { actions } = useActions(projectId);
     const [showDelDialog, setShowDelDialog] = useState(false);
+    const actionsCount = actions.filter(({ enabled }) => enabled).length;
     const navigate = useNavigate();
     return (
         <StyledContainer>
             <p>
                 Before you can delete a project, you must first archive all the
-                feature flags associated with it. Keep in mind that deleting a
-                project will permanently remove all the archived feature flags,
-                and they cannot be recovered once deleted.
+                feature flags associated with it
+                {isEnterprise() && automatedActionsEnabled
+                    ? 'and disable all actions that are in it'
+                    : ''}
+                .
             </p>
             <ConditionallyRender
-                condition={isEnterprise() && automatedActionsEnabled}
+                condition={featureCount > 0 || actionsCount > 0}
                 show={
-                    <p>
-                        Additionally, all configured actions for this project
-                        will no longer be executed as they will be permanently
-                        deleted.
-                    </p>
+                    <>
+                        <p>
+                            Currently there are{' '}
+                            <strong>{featureCount} feature flags active</strong>
+                        </p>
+                        <ConditionallyRender
+                            condition={
+                                isEnterprise() && automatedActionsEnabled
+                            }
+                            show={
+                                <p>
+                                    Currently there are{' '}
+                                    <strong>
+                                        {actionsCount} enabled actions
+                                    </strong>
+                                </p>
+                            }
+                        />
+                    </>
```

Comment:
I think this can be a bit weird in some edge cases:
 - featureCount === 0
 - actionsCount > 0
 - automatedActionsEnabled === false

We would only show "0 feature flags active", which I'm not sure is the intended behavior. I don't think it matters much, but I would pick one of:
 - Show zeroes: No longer care about the total `featureCount` and `actionsCount` totals, show them even if they are 0. The only condition to show the actions paragraph is `isEnterprise() && automatedActionsEnabled`
 - Don't show zeroes: Add a `ConditionallyRender` wrapper around each paragraph where we check the individual count for each along with any other conditions: `featureCount > 0` and `actionsCount > 0 && isEnterprise() && automatedActionsEnabled`